proto/cosmos/upgrade/v1beta1/upgrade.proto

```diff
@@ -40,6 +40,39 @@ message Plan {
   // If this field is not empty, an error will be thrown.
   google.protobuf.Any upgraded_client_state = 5
       [deprecated = true];
+
+  // Optional: Upgrade contains additional instructions for the devops or a hypervisor.
+  // App specific instructions are handled by the `info` attribute. Here we provide
+  // information such as pre-upgrade or post-upgrade commands.
+  UpgradeInstructions Upgrade = 6;
+}
+
+message UpgradeInstructions {
+  option (gogoproto.equal)            = true;
+
+  // If not empty, a shell command to be run by the upgrade manager or a hypervisor after shutting down
+  // the app and before running a new node.
+  string pre_run = 1;
+  // If not empty, a shell command to be run by the upgrade manager or a hypervisor after shutting down
+  // the app and after running a new app.
+  string post_run = 2;
+  // List of required assets to download. This follows the cosmovisor structure.
+  // SHOULD have only one entry per platform.
+  repeated Asset assets    = 3;
+  // Description contains additional information about the upgrade process. Can reference an external resource.
+  string description = 4;
```

Comment:
I think we can remove this `description` field.

If someone provides `UpgradeInstructions` in the `Plan` then the `info` field can then be used for this. If `UpgradeInstructions` aren't included in the `Plan` then this field wouldn't be there to populate.

I guess that if the `UpgradeInstructions` are included, then the `info` field is ignored (as far as automated processing goes).